pandas/_libs/tslibs/timezones.pyx

```diff
@@ -3,10 +3,13 @@ from datetime import (
     timezone,
 )
 
+from pandas.compat._optional import import_optional_dependency
+
 try:
     # py39+
     import zoneinfo
     from zoneinfo import ZoneInfo
+    import_optional_dependency("tzdata", errors="raise", min_version="2022.1")
```

Comment:
IIRC we should only check `tzdata` if the user doesn't have an OS that already has system timezones that are available.

```
import zoneinfo
from zoneinfo import ZoneInfo
try:
    ZoneInfo("UTC")
except zoneinfo.ZoneInfoNotFoundError:
    import_optional_dependency("tzdata", errors="warn", min_version="2022.1")
```

Also I think `errors="warn"` since `tzdata` is still optional, and the user may not necessarily use `ZoneInfo` objects when using pandas (since it's not the default yet)